specification/automanage/resource-manager/Microsoft.Automanage/preview/2021-03-30-preview/examples/listConfigurationProfileAssignmentIntentsBySubscription.json

```diff
@@ -0,0 +1,52 @@
+{
+  "parameters": {
+    "subscriptionId": "mySubscriptionId",
+    "vmName": "myVMName",
+    "api-version": "2021-03-30-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": [
```

Comment:
Response should be:
{
   "values": []
}

Reply:
Thank you! Resolved!